### ISSUE TITLE
Request: YacLibrary - Option to show covers in landscape format

### ISSUE THREAD

**Issue by FaceCrap** [YACReader/yacreader#396]
Since I'm running YAC (Library and Reader) on Windows, most all the comics I download are formatted in landscape view (including the cover). However, YACLibrary assumes that we're using Portrait monitors and shows all covers in portrait mode even if only landscape formatted covers exist, thus cutting off large portions of the original cover.

It would be very useful if there was an option to either choose or have the program detect orientation automatically whether covers should be shown in landscape or portrait mode. In auto mode, it is easy to detect orientation from the image dimensions. But a manual override option on comic level, where a comic has both a landscape and portrait version, would be even more useful.

**Comment by luisangelsm** [comment]
Yeah, supporting different aspect ratios is something I would like to add at some point.
